main.py

```
@@ -1,55 +1,43 @@
+)
+from tale_studio.openai_wrapper import DEFAULT_MODEL
+from tale_studio.recurrentgpt import RecurrentGPT
+from tale_studio.human_simulator import Human
+
+
+def main(
+    n_iter: int = 1,
+    out_file: str = "out.jsonl",
+    novel_type: str = "science fiction",
+    description: str = "",
+    model_name: str = DEFAULT_MODEL_NAME,
```

Ruff (0.14.4)

Unused function argument: `model_name`

(ARG001)